Common/FeatureFlags.swift

```diff
@@ -58,6 +60,20 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.simulatedCoreDataEnabled = false
         #endif
+
+        // Swift compiler config is inverse, since the default state is enabled.
```

Comment:
Or, you could be explicit and add it (along with its default) to `Loop/Loop.xcconfig`, no?  Also, If the default is "enabled", and you want to eliminate them, won't this code still include the models?  Or are you relying on overriding in `Loop/LoopOverride.xcconfig`?  (if so, shouldn't that be in this PR, too?). Forgive me if I'm missing something obvious.

Reply:
With the way we use `SWIFT_ACTIVE_COMPILATION_CONDITIONS` to define feature flag environment variables to change functionality in Tidepool Loop (as opposed to the default behavior of DIY Loop) we need to declare the environment variable in terms of what we *want* to happen different in Tidepool Loop. Thus, the use of `WALSH_INSULIN_MODEL_DISABLED` to indicate we want Tidepool Loop to *disable* the Walsh insulin model.

Then, rather than have the code be confusing where some FeatureFlags are "Enabled" and others are "Disabled" we just reverse the logic of the environment variable so all FeatureFlags properties use the "Enabled" prefix.

BTW, this is nothing new. I'm just copying how it is done for the existing FeatureFlags `FEATURE_OVERRIDES_DISABLED`, `NONLINEAR_CARB_MODEL_DISABLED`, and `REMOTE_OVERRIDES_DISABLED`.

Reply:
BTW, there is no way to turn *off* one of these environment variables compiler switches once it is turned on. (You'd have to somehow strip it from `SWIFT_ACTIVE_COMPILATION_CONDITIONS`, but I don't think you can do that in an `xcconfig` file.